### PR TITLE
Portfolio mobile version module 1 day 4

### PR DESCRIPTION
**Pull request by carlosmuerto**
- Add My work section 
- Add About me section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,6 +55,114 @@ <h1>Hi there I'm Carlos Vivas <br> <span class="highlight">I'm a software Develo
...
+						<li>css</li>
+						<li>JavaScript</li>
+						<li>html</li>
+					</ul>
+					<button disabled="disabled">See Project</button>
+				</div>
+			</article>
+
+			<article class="card">
+				<!-- <img class="card-img" src="assets/works/Img Plaholder.jpg" alt="preview"> -->
```
Comment (danielufeli) [inline index.html]: - [x] Kindly remove all commented code, so your codebase is clean and readable.

File: index.html [hunk 2]
```diff
@@ -55,6 +55,114 @@ <h1>Hi there I'm Carlos Vivas <br> <span class="highlight">I'm a software Develo
...
+				<div class="card-body">
+					<h3>Multi-Post Stories Gain+Glory 02</h3>
+					<ul>
+						<li>Ruby on rails</li>
+						<li>css</li>
+						<li>JavaScript</li>
+						<li>html</li>
+					</ul>
+					<button disabled="disabled">See Project</button>
```
Comment (danielufeli) [inline index.html:91]: - [ ] Nice working on your `My Recent Works Section`, kindly change the color of the `See Project Button` so it matches what is in the Figma Template.

### Figma Design
fig123

### Your Design 
my123

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @carlosmuerto, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- All linters are passing ✔️
- Followed Github flow ✔️
- PR has a good description summary ✔️
- Clean Design ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (carlosmuerto) [reply index.html:91]: ### I'm using an alternative colour palette. the blue one

#### palette
imagen

#### buttons
imagen

**Review by codecaiine — APPROVED** [review]
Hi @carlosmuerto , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
All linters are passing :heavy_check_mark:
Followed Github flow :heavy_check_mark:
PR has a good description summary :heavy_check_mark:
Clean Design :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**